pkg/operation/cloudbotanist/gcpbotanist/gcpbotanist.go

```diff
@@ -68,9 +71,16 @@ func New(o *operation.Operation, purpose string) (*GCPBotanist, error) {
 		return nil, err
 	}
 
+	// TODO: context.TODO() will be replaced by real context as soon as it is available
+	client, err := gcp.NewClient(context.Background(), serviceAccountJSON, project)
```

Comment:
```suggestion
	client, err := gcp.NewClient(context.TODO(), serviceAccountJSON, project)
```

and remove above comment